### DISCUSSION TITLE
Definition of the variable "Change in open-circuit voltage [V]"

### DISCUSSION THREAD

**Discussion by JayYDG** [pybamm-team/PyBaMM#3305] [Q&A] [answered]
I just start to play with the pybamm with the SPMe model. Just wonder what is the definition of "Change in open-circuit voltage [V]"? And where this variable is defined?

**Comment by TomTranter** [comment, marked as answer]
Defined here 
https://github.com/pybamm-team/PyBaMM/blob/840fb1312029bcd3d4c7c2be2798e9ee9f0fbc08/pybamm/models/full_battery_models/base_battery_model.py#L1247
It's the difference in ocv at a certain SOC and the initial ocv. You don't need to worry about it too much. Can be used to build equivalent circuit type models

**Reply by JayYDG** [reply]
Thank you for the quick response.
